omop/cdm_reference.qmd

```diff
@@ -17,6 +17,7 @@ library(duckdb)
 library(dplyr)
 library(omock)
 library(CDMConnector)
+library(omopgenerics) # to be removed after CDMConnector > 2.2.0
```

Comment:
Does this mean waiting to release the book until cdmconnector 2.2 is released? I would just remove the comment and leave as is for the first release